coda/templates/mdstore/node_status.html

```diff
@@ -38,6 +41,7 @@ <h3>
 				<td> <i class="icon-hdd"></i> {{ status.node.node_size|filesizeformat }} </td>
 				<td> <i class="icon-hdd"></i> {{ status.available|filesizeformat }} </td>
 				<td> <i class="icon-dashboard"></i> {{ status.filled }}% </td>
+                                <td> <i class="icon-calendar"></i> {{ status.node.get_status_display }} </td>

```

Comment:
Looks like alignment is off here

Reply:
I'll fix it.

Reply:
The issue is that this line is using spaces for the leading whitespace, while those around it are using tabs. We should be consistent.

Reply:
Ah, I see. I'll fix it. Thanks!